src/components/cc-pricing-product-consumption/cc-pricing-product-consumption.js

```diff
@@ -349,25 +349,28 @@ export class CcPricingProductConsumption extends withResizeObserver(LitElement)
         <cc-loader></cc-loader>
       ` : ''}
 
-      ${this.product.state === 'loaded' ? this._renderLoaded() : ''}
+      ${this.product.state === 'loaded' ? this._renderLoaded(this.product.sectionList) : ''}
     `;
   }
 
-  _renderLoaded () {
+  /**
+   * @param {PricingSection[]} sectionList
+   */
+  _renderLoaded (sectionList) {
     const bodyClasses = {
       'body--big': (this._size > BREAKPOINT),
       'body--small': (this._size <= BREAKPOINT),
     };
 
-    const someNullishIntervals = this.product.sectionList
-      ?.some(({ intervals }) => intervals == null) ?? true;
-    const everyQuantityAtZero = this.product.sectionList
-      ?.map(({ type }) => this._simulator.getQuantity(type))
-      ?.every((quantity) => quantity === 0) ?? true;
+    const someNullishIntervals = sectionList
+      .some(({ intervals }) => intervals == null) ?? true;
+    const everyQuantityAtZero = sectionList
+      .map(({ type }) => this._simulator.getQuantity(type))
+      .every((quantity) => quantity === 0) ?? true;
```

Comment:
todo: `.some()` and `.every()` always return a boolean, no need for the `?? true`